pages/addCommunity.tsx

```diff
@@ -64,6 +65,9 @@ const AddCommunity = (): JSX.Element => {
     }, 10000);
   };
   const validateForm = () => {
+    // if (title.length > 6) {
```

Comment:
Don't leave commented code in the codebase unless theres a clear reason that is documented. For example:
```
// DONT REMOVE: This will be used again when x is ready
// if (title.length .....
```